src/java/com/unifina/signalpath/remote/Http.java

```diff
@@ -35,6 +35,12 @@
 	private StringParameter URL = new StringParameter(this, "URL", "");
 	private MapParameter headers = new MapParameter(this, "headers");
 	private MapParameter queryParams = new MapParameter(this, "params");
+	private DataTypeParameter dataType = new DataTypeParameter(this, "dataType", "auto", Arrays.asList(
+		new PossibleValue("auto", null),
+		new PossibleValue("binary", "binary"),
+		new PossibleValue("text", "text"),
+		new PossibleValue("json", "json")
```

Comment:
Should we store content type here also?
Like: `new DataTypePossibleValue("label", "content/type", null)`
Related to my comment down below